data/json/npcs/meteorologist.json

```
@@ -0,0 +1,308 @@
+[
+  {
+    "type": "npc",
+    "id": "meteorologist",
+    "class": "meteorologist",
+    "name_suffix": "meteorologist",
+    "attitude": 0,
+    "mission": 7,
+    "chat": "meteorologist_1",
+    "faction": "no_faction",
+    "mission_offered": "MISSION_meteorologist_1"
+  },
+  {
+    "type": "npc_class",
+    "id": "meteorologist",
+    "name": { "str": "Meteorologist" },
+    "job_description": "Meteorologist.",
+    "common": false,
+    "bonus_str": { "rng": [ -2, 2 ] },
+    "bonus_dex": { "rng": [ -2, 2 ] },
+    "bonus_int": { "rng": [ 2, 5 ] },
+    "bonus_per": { "rng": [ -1, 3 ] },
+    "worn_override": "meteorologist_worn",
+    "traits": [ { "group": "NPC_starting_traits" }, { "group": "Appearance_demographics" }, { "trait": "BGSS_METEOROLOGIST" } ],
+    "skills": [
+      { "skill": "ALL", "level": { "mul": [ { "one_in": 3 }, { "sum": [ { "dice": [ 2, 2 ] }, { "constant": -4 } ] } ] } },
+      { "skill": "computer", "bonus": { "rng": [ 2, 3 ] } },
+      { "skill": "electronics", "bonus": { "rng": [ 3, 6 ] } },
+      { "skill": "chemistry", "bonus": { "rng": [ 2, 3 ] } }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "meteorologist_worn",
+    "subtype": "collection",
+    "entries": [
+      { "item": "briefs" },
+      { "item": "coat_lab" },
+      { "item": "dress_shirt" },
+      { "item": "pants" },
+      { "item": "socks" },
+      { "item": "boots" },
+      { "item": "gloves_rubber" },
+      { "item": "wristwatch" },
+      { "item": "glasses_safety" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": [ "meteorologist_1" ],
```

We normally preface these ID's with something like `TALK_`, so that they don't cross with other IDs and to make it easier to read everything. So this would be 
```suggestion
    "id": [ "TALK_meteorologist_1" ],
```
and you'd wan tto go through and change the other pointers and IDs to match.

---

Done.